flymake-sqlfluff.el

```diff
@@ -115,8 +115,8 @@
 
 (defun flymake-sqlfluff-process-item (item)
   "Build a flymake diagnostic using ITEM data."
-  (let* ((line (gethash "line_no" item))
-         (column (gethash "line_pos" item))
+  (let* ((line (gethash "start_line_no" item))
+         (column (gethash "start_line_pos" item))
```

Comment:
Can you add a default so we can still support version 2? Something like this

```emacs-lisp
(gethash "start_line_no" item (gethash "line_no" item))
```

Reply:
Okay, added